src/handlers/meta_transaction_handlers.ts

```diff
@@ -39,6 +47,134 @@ export class MetaTransactionHandlers {
         this._metaTransactionService = metaTransactionService;
     }
 
+    /**
+     * Handler for the /meta_transaction/v2/quote endpoint
+     */
+    public async getV2QuoteAsync(req: express.Request, res: express.Response): Promise<void> {
```

Comment:
how much shared code is there between this logic and the v1QuoteAsync logic? Same for v1PriceAsync?

Reply:
Assuming you're talking about handler level, parsing of incoming request and get meta-transaction v2 quote are different. The majority of the shared code on handler level is error handling

Reply:
Got it. I will do another refactoring after merging this scaffolding PR. Want to limit the scope of the scaffolding